Description: Hardcode version in pkg-config file
 Since we do not run autoreconf in Maemo,
 we simply hardcode the version in libssh2.pc.in
 instead of using fix-version-in-pc.patch.
Author: Graham Inggs <graham@nerve.org.za>
Forwarded: not-needed
Last-Update: 2015-03-01
--- a/libssh2.pc.in
+++ b/libssh2.pc.in
@@ -10,7 +10,7 @@
 Name: libssh2
 URL: http://www.libssh2.org/
 Description: Library for SSH based connunication
-Version: @VERSION@
+Version: 1.2.8
 Libs: -L${libdir} -lssh2 @LDFLAGS@ @LIBS@
 Libs.private: @LIBS@
 Cflags: -I${includedir}
